### DISCUSSION TITLE
Slanted bicrystal Grain boundary

### DISCUSSION THREAD

**Discussion by Leni-Yeo** [idaholab/moose#20168] [Q&A Modules: Phase field] [answered]
Good afternoon, 

I hope that you are doing well.

I was wondering if there were any way to have an initial condition with a slanted boundary between two crystals; with an inclination at various degrees, but still able to use grain tracker. I have been trying to apply the various Phase Filed initial conditions from MOOSE to get that, but I keep failing. 

Thank you in advance for the help.

**Comment by SudiptaBiswas** [comment, marked as answer]
The short answer probably is that no, we don't have a way to specify GB angle for bicrystal structure. Having said that,  if you create a bicrystal with any of the polycrystal initial conditions, you will be able to generate a slanted GB (I have done this with Voronoi IC). You might not have precise control over the angle though. Alternatively, you can derive from https://mooseframework.inl.gov/source/ics/BoundingBoxIC.html and create a new initial condition.

**Reply by Leni-Yeo** [reply]
I see. I will try as you suggested. 
Thank you for the answer.
